tutorials/back-up-postgresql-barman/index.mdx

```diff
@@ -269,7 +269,7 @@ Your database has now successfully migrated to your new database server.
 
 ### Automatizing Backups
 
-It is possible to run Barman automatically to back up the content of your database regularly. This can be done easily by setting up a cron job on `pgsql-backup`.
+It is possible to run Barman automatically to back up the content of your database regularly. This can be done easily by setting up a CronJob on `pgsql-backup`.
```

Comment:
```suggestion
It is possible to run Barman automatically to back up the content of your database regularly. This can be done easily by setting up a cron job on `pgsql-backup`.
```